src/errors.rs

```diff
@@ -17,11 +16,11 @@ pub enum Error<E> {
     ///
     /// Typically this indicates a faulty implementation of `serde::Serialize` or
     /// `serde::Deserialize`.
-    ClientDeserialize(bincode::serde::DeserializeError),
+    ClientDeserialize(::bincode::Error),
```

Comment:
Yeah I was wondering if you wanted to do that. I think it's good to eliminate as well.

I removed both `Error::ServerDeserialize` and `WireError::ServerDeserialize`.